assignments/this.js

```diff
@@ -1,26 +1,55 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+ *1. Window Binding - binds to the window as the outermost bindable when not given what to bind too
+* 2. Explicit Binding - binds to what it is told to do
+* 3. New Binding - binds to an object that is to be created
+* 4. Implicit Binding - binds to what makes the most sense
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
 
 // code example for Window Binding
+  console.log (this);
```

Comment:
Also a simple function for window binding:  

function pickColor(color) {
    console.log(this);
    return color;
  }
  pickColor("blue");